solana/rpc/api.py

```diff
@@ -42,6 +42,46 @@ def get_balance(self, pubkey: Union[PublicKey, str]) -> RPCResponse:
         """
         return self._provider.make_request(RPCMethod("getBalance"), str(pubkey))
 
+    def get_account_info(self, pubkey: Union[PublicKey, str]) -> RPCResponse:
```

Comment:
Please add request options.

Example: https://github.com/michaelhly/solana-py/blob/23a159ec75e3434f9ee473034d3d99937a710381/solana/rpc/api.py#L520.

Reply:
Added.

Reply:
I just added the support for the commitment option. Could you pull master and add that as well. Thank you!

Reply:
Done!